v2/emailpassword/common-customizations/get-user-info.mdx

```diff
@@ -574,30 +586,58 @@ function Dashboard(props: any) {
 import Session from 'supertokens-auth-react/recipe/session';
 
 async function getUserInfo() {
-      if (await Session.doesSessionExist()) {
-            let userId = await Session.getUserId();
-            let accessTokenPayload = await Session.getAccessTokenPayloadSecurely();
-      }
+    if (await Session.doesSessionExist()) {
+        let userId = await Session.getUserId();
+        let accessTokenPayload = await Session.getAccessTokenPayloadSecurely();
+    }
 }
 ```
 
 </TabItem>
+
 </FrontendReactContextSubTabs>
+
+</TabItem>
+
+<TabItem value="angular">
+
+```tsx
+import Session from 'supertokens-auth-react/recipe/session';
+
```

Comment:
this can use copy tabs and copy from the without react context sub tab of the react tab?